frontend/src/scenes/App.js

```diff
@@ -10,6 +10,7 @@ import { ToastContainer, Slide } from 'react-toastify'
 import { Sidebar } from '~/layout/Sidebar'
 import { TopContent } from '~/layout/TopContent'
 import { SendEventsOverlay } from '~/layout/SendEventsOverlay'
+import { OnboardingWizard } from '~/scenes/onboarding/onboardingWizard'
```

Comment:
Q: Would it make sense to load this + its dependencies in separate chunks as well?

Not a huge issue, but from what I can tell from the code this does make the JS payload larger on every page with the instructions for every platform

Reply:
Yes, good call!